### ISSUE TITLE
Update SDK to Keycloak 26

### ISSUE THREAD

**Issue by JonasDev17** [NikiforovAll/keycloak-authorization-services-dotnet#153]
With Keycloak 26 organizations are officially supported which can be used for multi tenant applications.
And with version 26.0.2 the api documentation has now been updated to include the organization endpoints.
https://www.keycloak.org/docs-api/latest/rest-api/index.html#_organizations
Would be nice if the SDK would support these endpoints.

**Comment by NikiforovAll** [comment]
Yes, I would like to do that. But, I think the auto-generated client is the way to go.

**Comment by NikiforovAll** [comment]
Closes via: #154
